daliuge-translator/dlg/translator/tool_commands.py

```diff
@@ -301,8 +301,11 @@ def cwl(parser, args):
     from ..dropmake.cwl import create_workflow
 
     # write to file
-    with _open_o(opts.output, "wb") as f:
-        create_workflow(pgt, "workflow.cwl", f)
+    try:
+        with _open_o(opts.output, "wb") as f:
+            create_workflow(pgt, "workflow.cwl", f)
+    except Exception as e:
+        logger.error(e)
```

Comment:
Looking at the rest of the command line tools there's no other specific treatment of errors in any of them (which default will print the full exception stacktrace). Maybe we can leave this one like that too? (i.e., no need to modify this at all).

Reply:
I agree, better to be consistent.